src/server/game/AI/PlayerAI/PlayerAI.cpp

```diff
@@ -196,6 +1280,31 @@ void SimpleCharmedPlayerAI::UpdateAI(const uint32 /*diff*/)
             else
                 AttackStart(target);
         }
+
+        if (me->IsStopped() && !me->HasUnitState(UNIT_STATE_CANNOT_TURN))
+        {
+            float targetAngle = me->GetAngle(target);
+            if (fabs(me->GetOrientation() - targetAngle) > 0.4f)
+            {
+                me->SetFacingTo(targetAngle);
+                me->SetOrientation(targetAngle);
```

Comment:
Is it neccessary to use SetFacing and SetOrientation? Shouldn't it be ok with just one?